src/components/network/selection-creation-panel.tsx

```diff
@@ -37,33 +37,27 @@ import { fetchDirectoryElementPath } from '@gridsuite/commons-ui';
 import CircularProgress from '@mui/material/CircularProgress';
 import FolderOutlined from '@mui/icons-material/FolderOutlined';
 
-interface IFilterSelection {
-    [SELECTION_NAME]: string | null;
-    [NAME]: string;
-    equipmentType: string | null;
-    [SELECTION_TYPE]: string | null;
-}
-
 const formSchema = yup
     .object()
     .shape({
-        [SELECTION_NAME]: yup.string().nullable(),
         [NAME]: yup.string().required(),
-        equipmentType: yup.string().required(),
-        [SELECTION_TYPE]: yup.string().required(),
+        [SELECTION_TYPE]: yup.string().nullable().required(),
+        [EQUIPMENT_TYPE_FIELD]: yup.string().nullable().required(),
```

Comment:
i don't know if it make sense to have a nullable required string ?
i suggest :` [SELECTION_TYPE]: yup.string().min(1).required(),` @klesaulnier  do you agree ?

Reply:
nullable().required() is ok
it means your field can be null, but is required 

If you don't add "nullable", an error "field can't be null" will be thrown
Here, we only want an error "field is required"

It's only ok here because the field is an autocomplete field, for a "TextInput", we should not do it